services/storage/handlers/packages.js

```diff
@@ -0,0 +1,509 @@
+'use strict';
+
+const { Response } = require('node-fetch');
+const { markdown } = require('markdown');
```

Comment:
This particular markdown lib is very abandoned. It might make sense to use something like markdown-it, which is both maintained and super pluggable.

Reply:
I was waiting to use `marky-markdown`, which is a Known Good package for rendering markdown; _however_ it brings along oniguruma so I wanted to wait until we had separated package publishes out into a worker process.

Reply:
Added [a ticket](https://github.com/entropic-dev/entropic/issues/214) so this doesn't get lost.